code/datums/craft/recipes/tools.dm

```diff
@@ -18,7 +18,7 @@
 	result = /obj/item/tool/knife/shiv
 	steps = list(
 		list(/obj/item/material/shard, 1, "time" = 30),
-		list(QUALITY_ADHESIVE, 15, 70)
+		list(/obj/item/stack/medical/bruise_pack/handmade, 1, "time" = 10)
```

Comment:
What? Adhesive is duct tape (and web tape and what else), it's very easy to get.

Reply:
I had a discussion with garrulous about this, about what the "starting step" should be for crafting the makeshift tools. We came to an agreement that the shiv should be the most easily available tool and thought it was weird you needed specifically duct tape (which isnt that hard to find, true) to make a shiv, perhaps the most basic tool possible. Changing this to handmade brute packs makes it so that you can build a shiv in almost any situation, which is what it should be IMO.

Reply:
Edit it so people can make it with web tape, which is tier 1 duct tape.

Reply:
That's a decent compromise but id still rather it just be basic cloth, since that's more or less what a shiv with. Will wait to see what the inflamed woodland creature has to say

Reply:
Is adhesive easy to get? With a roundstart autolathe and roundstart tool disk it is, but that kinda defeats the point of makeshift tools. There is always RNG trash find, but rng is rng. And making web tape needs a thing of spider web, which means either dealing with spiders or some measure of rng

Reply:
You can start with it but I'm envisioning perma-brig esque scenarios where your things are taken from you and you're unable to get proper replacements. IE a desperate situation instead of a normal round where yes, adhesive would be very easy to acquire.

Reply:
It has blue tape on sprite, not cloth.

Reply:
Also almost literally every other multitool asks for `QUALITY_ADHESIVE`. I do not see an issue shiv requiring it as well.

Reply:
Changed it back to taking adhesive.